node/test/lib/get-process-function.js

```diff
@@ -35,9 +35,10 @@ module.exports = async (basename, coreConfig, options) => {
       },
       Timeout: 15,
     },
-    ensurePlainObject(options.baseLambdaConfiguration)
+    ensurePlainObject(options.baseLambdaConfiguration || {})
```

Comment:
This I think should stay as mandatory option

Reply:
Ahh yeah my bad 🤦‍♂️